build/web-features.ts

```diff
@@ -1,16 +1,37 @@
 import { features } from "web-features";
+import { computeBaseline } from "compute-baseline";
 
-export function getWebFeatureStatus(...bcdKeys: string[]) {
-  if (bcdKeys.length === 0) {
-    return;
+import type { Doc } from "../libs/types/document.js";
+
+export function addBaseline(doc: Partial<Doc>) {
+  if (doc.browserCompat && !doc.mdn_url?.includes("/docs/MDN/")) {
+    if (doc.browserCompat.length !== 1) {
+      return;
+    }
```

Comment:
Nit: Could be combined.

```suggestion
  if (!doc.browserCompat || doc.browserCompat.length !== 1 || doc.mdn_url?.includes("/docs/MDN/")) {
    return;
  }
```